node-manager/Cargo.toml

```diff
@@ -10,8 +10,11 @@ crate-type = ["cdylib"]
 cfg-if = "1.0.0"
 wasm-bindgen = "0.2.83"
 bip39 = { version = "1.0.1" }
-bitcoin = "0.29.1"
+bitcoin = "0.28.1"
+bdk = { git = "https://www.nakamoto.codes/BitcoinDevShop/bdk", branch="feat/use-external-esplora-client", default-features = false}
```

Comment:
once https://github.com/bitcoindevkit/bdk/pull/770 is merged, we should update to that